pkgs/tools-cli/src/main.zig

```diff
@@ -0,0 +1,219 @@
+const std = @import("std");
+const enr = @import("enr");
+const build_options = @import("build_options");
+const simargs = @import("simargs");
+
+const ToolsArgs = struct {
+    help: bool = false,
+    version: bool = false,
+
+    __commands__: union(enum) {
+        enrgen: ENRGenCmd,
+
+        pub const __messages__ = .{
+            .enrgen = "Generate a new ENR (Ethereum Node Record)",
+        };
+    },
+
+    pub const __shorts__ = .{
+        .help = .h,
+        .version = .v,
+    };
+
+    pub const __messages__ = .{
+        .help = "Show help information",
+        .version = "Show version information",
+    };
+
+    const ENRGenCmd = struct {
+        sk: []const u8,
+        ip: []const u8,
+        udp: u16,
```

Comment:
i think instead of `udp` field we have `quic` field, see https://github.com/leanEthereum/leanSpec/blob/main/docs/client/networking.md

Reply:
Oh, it seems the spec is not good here, it doesn't need a quic field. there is already a standard udp field there.  Combine secp256k1, ip, udp, it can generate a ipv4 multiaddr. This is already used in beacon chain, i think we need change the spec here.

Reply:
ok, I saw here https://github.com/sigp/lighthouse/blob/stable/beacon_node/lighthouse_network/src/discovery/enr_ext.rs#L9, let me add `quic` field.